### PR TITLE
Add returns endpoint

### PR DESCRIPTION
**Pull request by jacobdekeizer**
Hi,

Thank you for providing this package for the new bol.com api version.
I've added the returns endpoints with example file.

### CODE REVIEW

File: src/Model/ReturnItem.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php
+namespace  Picqer\BolRetailer\Model;
+
+use DateTime;
+
+/**
+ * @property int           $rmaId
```
Comment (kleiram) [inline src/Model/ReturnItem.php:7]: Can you add documentation about these fields to the property annotations?

File: src/ReturnItem.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        }, $returns);
+    }
+
+    /**
+     * Handle the return.
+     *
+     * @param int    $rmaId            The RMA id that identifies this particular return.
+     * @param string $handlingResult   The return item handling type.
+     * @param int    $quantityReturned The amount of items returned.
+     * @return Model\ProcessStatus
```
Comment (kleiram) [inline src/ReturnItem.php:62]: There should be a new line between the `@param` and `@return` annotations.

**Comment by jacobdekeizer** [comment]
@kleiram Fixed :)

**Comment by kleiram** [comment]
@jacobdekeizer Thank you very much! 👏

**Pull request merged**